ReactAndroid/src/main/java/com/facebook/react/modules/datepicker/DatePickerDialogFragment.java

```diff
@@ -53,8 +53,13 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
     final int month = c.get(Calendar.MONTH);
     final int day = c.get(Calendar.DAY_OF_MONTH);
 
-    final DatePickerDialog dialog =
-        new DismissableDatePickerDialog(activityContext, onDateSetListener, year, month, day);
+    DatePickerDialog dialog;
+    if (args != null && args.containsKey(DatePickerDialogModule.ARG_SPINNERMODE) && args.getBoolean(DatePickerDialogModule.ARG_SPINNERMODE)) {
```

Comment:
do we need `args.containsKey(DatePickerDialogModule.ARG_SPINNERMODE)`?
`args.getBoolean(DatePickerDialogModule.ARG_SPINNERMODE)` defaults to false